synapseclient/models/protocols/wikipage_protocol.py

```diff
@@ -0,0 +1,345 @@
+    )
+
+
+class WikiOrderHintSynchronousProtocol(Protocol):
+    """Protocol for the methods of the WikiOrderHint class that have synchronous counterparts
+    generated at runtime.
+    """
+
+    def store(
+        self,
+        *,
+        synapse_client: Optional["Synapse"] = None,
+    ) -> "WikiOrderHint":
+        """
+        Store the order hint of a wiki page tree.
+        Arguments:
+            synapse_client: Optionally provide a Synapse client.
+        Returns:
+            The updated WikiOrderHint object for the entity.
+
+        Example: Set the WikiOrderHint for a project
+            This example shows how to set a WikiOrderHint for existing wiki pages in a project.
+            The WikiOrderHint is not set by default, so you need to set it explicitly.
+            from synapseclient import Synapse
+            from synapseclient.models import (
+                Project,
+                WikiOrderHint,
+            )
+            syn = Synapse()
+            syn.login()
+
+            project = Project(name="My uniquely named project about Alzheimer's Disease").get()
+            # get the WikiOrderHint for the project
+            wiki_order_hint = WikiOrderHint(owner_id=project.id).get()
+
+            wiki_order_hint.id_list = [
+                root_wiki_page.id,
+                wiki_page_1.id,
+                wiki_page_3.id,
+                wiki_page_2.id,
+                wiki_page_4.id,
+            ]
+            wiki_order_hint.store()
+            print(wiki_order_hint)
+
+        Example: Update the WikiOrderHint for a project
+            This example shows how to update a WikiOrderHint for existing wiki pages in a project.
+            wiki_order_hint.id_list = [
+                root_wiki_page.id,
+                wiki_page_1.id,
+                wiki_page_2.id,
+                wiki_page_3.id,
+                wiki_page_4.id,
+            ]
+            wiki_order_hint.store()
+            print(wiki_order_hint)
+        """
+        return self
+
+    def get(
+        self,
+        *,
+        synapse_client: Optional[Synapse] = None,
+    ) -> "WikiOrderHint":
+        """
+        Get the order hint of a wiki page tree.
+        Arguments:
+            synapse_client: Optionally provide a Synapse client.
+        Returns:
+            A WikiOrderHint object for the entity.
+
+        Example: Get the WikiOrderHint for a project
+            This example shows how to get a WikiOrderHint for existing wiki pages in a project.
+            from synapseclient import Synapse
+            from synapseclient.models import (
+                Project,
+                WikiOrderHint,
+            )
+            syn = Synapse()
+            syn.login()
+
+            project = Project(name="My uniquely named project about Alzheimer's Disease").get()
+            # get the WikiOrderHint for the project
+            wiki_order_hint = WikiOrderHint(owner_id=project.id).get()
+
+            print(wiki_order_hint)
```

Comment:
The formatting of every example that you've added is a bit broken:

<img width="1085" height="404" alt="Image" src="https://github.com/user-attachments/assets/880be276-f908-4f63-9808-466d39c5866b" />


```suggestion
        Example: Get the WikiOrderHint for a project
            This example shows how to get a WikiOrderHint for existing wiki pages in a project.
            
            ```python
            from synapseclient import Synapse
            from synapseclient.models import (
                Project,
                WikiOrderHint,
            )
            syn = Synapse()
            syn.login()

            project = Project(name="My uniquely named project about Alzheimer's Disease").get()
            # get the WikiOrderHint for the project
            wiki_order_hint = WikiOrderHint(owner_id=project.id).get()

            print(wiki_order_hint)
            ```
```

The code needs to be wrapped in backticks

Reply:
Will do. @BryanFauble qq. how can I render the docs locally?